integration/daemon/daemon_test.go

```diff
@@ -400,3 +401,23 @@ func testLiveRestoreVolumeReferences(t *testing.T) {
 		runTest(t, "no")
 	})
 }
+
+func TestDaemonDefaultBridgeWithFixedCidrButNoBip(t *testing.T) {
+	skip.If(t, runtime.GOOS == "windows")
+
+	bridgeName := "ext-bridge1"
+	d := daemon.New(t, daemon.WithEnvVars("DOCKER_TEST_CREATE_DEFAULT_BRIDGE="+bridgeName))
+	defer func() {
+		d.Stop(t)
+		d.Cleanup(t)
+	}()
+
+	defer deleteInterface(bridgeName)
+	d.StartWithBusybox(t, "--bridge", bridgeName, "--fixed-cidr", "192.168.130.0/24")
+}
+
+func deleteInterface(ifName string) {
+	icmd.RunCommand("ip", "link", "delete", ifName)
+	icmd.RunCommand("iptables", "-t", "nat", "--flush")
+	icmd.RunCommand("iptables", "--flush")
```

Comment:
The results of running these commands should probably be asserted on, at least so that any nonzero exits are logged along with the failing commands' output.

Reply:
I'm not sure if that's really needed. `ip link delete` would fail if the daemon didn't start properly. OTOH if those commands don't run properly they could leave the host in such a state that some tests could become flakes :thinking: